stock_orderpoint_uom/models/procurement_group.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# (http://www.opensourceintegrators.com)
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ProcurementGroup(models.Model):
+    _inherit = "procurement.group"
+
+    @api.model
+    def run(self, procurements):
```

Comment:
```suggestion
    def run(self, procurements, raise_user_error=True):
```
run is a function of core